google/cloud/bigtable/cluster.py

```diff
@@ -356,4 +378,8 @@ def _to_pb(self):
             serve_nodes=self.serve_nodes,
             default_storage_type=self.default_storage_type,
         )
+        if self._kms_key_name:
```

Comment:
same question here, what does this look like when kms key is not set? Is there a default view?

Reply:
When I look at the PBs returned from the service, I'm finding that it's omitting `encryption_config` altogether if using Google default encryption, so this creates a PB the same as we get them from the service.